doc/admin-guide/files/records.config.en.rst

```diff
@@ -3658,7 +3658,8 @@ HTTP/2 Configuration
 
    Specifies how many number of PRIORITY frames |TS| receives for a minute at maximum.
    Clients exceeded this limit will be immediately disconnected with an error
-   code of ENHANCE_YOUR_CALM.
+   code of ENHANCE_YOUR_CALM. If this is set to 0, the limit logic is disabled.
+   This limit only will be enforced if
```

Comment:
will be enforced if ... ?

Reply:
sigh, the last sentence got trapped on my editor's buffer, Solved!